styles.css

```diff
@@ -272,4 +285,60 @@ button:active {
   padding-bottom: 5%;
 }
 
+/* contactme */
+
+#contactme {
+  border-top-left-radius: 60px;
+  background-color: #6070ff;
+  background-image: url(./Images/background/contact\ form\ shapes-mobile.svg);
+  background-size: 62%;
+  background-position: 100% 15%;
+  background-repeat: no-repeat;
+  padding: 20% 0 40% 0;
+}
```

Comment:
Your form is quite good. However, it's part of the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/91668f1d04ab043c8fac118a333ea1d5695b54b3/portfolio/3_contact_form.md?plain=1#L49) to use flexbox to lay out the elements. Kindly implement this too.